motor/providers/microsoft/auth.go

```diff
@@ -25,7 +25,7 @@ func (p *Provider) GetTokenCredential() (azcore.TokenCredential, error) {
 		case vault.CredentialType_pkcs12:
 			certs, privateKey, err := azidentity.ParseCertificates(p.cred.Secret, []byte(p.cred.Password))
 			if err != nil {
-				return nil, errors.Wrap(err, "could not parse certificate")
+				return nil, errors.Wrap(err, "could not parse provided certificate")
```

Comment:
Do we know the path of the certificate at this point? This might be helpful to the user.

Reply:
good point, we know this (via the param that is passed in) but this will take a more wiring to get it done.

i will look into it

Reply:
done, ive updated the PR description with the new error message that also shows the cert path